MMM-WienerLinien.js

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable block-scoped-var */
+/* eslint-disable func-names */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable no-var */
+/* eslint-disable vars-on-top */
```

Comment:
what impact does this have?

Reply:
This just disables some linting warnings, you can get rid of it, then you will see a few warnings